server/auth.js

```diff
@@ -56,14 +63,36 @@ module.exports = (() => {
     login: (req, res) => strategy.login(req, res),
     // local sign-in
     signin: (req, res, done) => passport.authenticate('local-signin', (err, user) => {
-      if (err) {
-        return res.status(401).json({ status: 'error', message: err.statusText });
-      }
-      if (!user) {
-        return res
-          .status(401)
-          .json({ status: 'error', message: 'Invalid Login' });
+      if (err && err.errors && err.errors[0] && err.errors[0]) {
+        const {
+          detail: errDetail,
+          status: errStatus,
+        } = err.errors[0];
+        let responseMessage = '';
+        const responseStatus = errStatus || 500;
+
+        switch (errDetail) {
+          case 'Database unavailable':
+            responseMessage = 'There was an issue with the login. Please, try again later.';
+            break;
+          case 'Invalid email or password':
+            responseMessage = 'Invalid Login';
+            break;
+          default:
+            responseMessage = errDetail || 'Something went wrong.';
+        }
+
+        return res.status(responseStatus).json({
+          status: 'error',
+          statusCode: responseStatus,
+          message: responseMessage,
+        });
       }
+      // if (!user) {
```

Comment:
I think we can get rid of this, as this is kind of an error and can be handled above.

Reply:
depends on what case this covers.